renderdoc/driver/gl/wrappers/gl_emulated.cpp

```diff
@@ -314,6 +314,26 @@ void APIENTRY _glNamedFramebufferRenderbufferEXT(GLuint framebuffer, GLenum atta
   GL.glFramebufferRenderbuffer(eGL_DRAW_FRAMEBUFFER, attachment, renderbuffertarget, renderbuffer);
 }
 
+void APIENTRY _glInvalidateNamedFramebufferData(GLuint framebuffer, GLsizei numAttachments,
+                                                const GLenum *attachments)
+{
+  bool nativeSupport = IsGLES ? GLCoreVersion >= 30 : GLCoreVersion >= 43;
+  if(nativeSupport || HasExt[ARB_invalidate_subdata])
```

Comment:
If the extension macro is fixed (see other comment) this `nativeSupport` will not be needed as the extension check will account for core version.